cmd/swarm-rafttool/dump.go

```diff
@@ -84,17 +85,38 @@ func dumpWAL(swarmdir, unlockKey string, start, end uint64, redact bool) error {
 
 	for _, ent := range walData.Entries {
 		if (start == 0 || ent.Index >= start) && (end == 0 || ent.Index <= end) {
-			fmt.Printf("Entry Index=%d, Term=%d, Type=%s:\n", ent.Index, ent.Term, ent.Type.String())
+			if format == "text" {
+				fmt.Printf("Entry Index=%d, Term=%d, Type=%s:\n", ent.Index, ent.Term, ent.Type.String())
+			}
 			switch ent.Type {
 			case raftpb.EntryConfChange:
+
 				cc := &raftpb.ConfChange{}
 				err := proto.Unmarshal(ent.Data, cc)
 				if err != nil {
 					return err
 				}
-
-				fmt.Println("Conf change type:", cc.Type.String())
-				fmt.Printf("Node ID: %x\n\n", cc.NodeID)
+				switch format {
+				case "text":
+					fmt.Println("Conf change type:", cc.Type.String())
+					fmt.Printf("Node ID: %x\n\n", cc.NodeID)
+					fmt.Println()
+				case "json":
+					err := json.NewEncoder(os.Stdout).Encode(struct {
```

Comment:
the convention for CLI json output is pretty-printed (one field per line).  I'll take a look at how `docker inspect` does this and make it so.